docs/conf.py

```diff
@@ -34,6 +34,9 @@
 autorunrecord_line_replace = [
     # trailing space removal
     (r'[ ]+$', ''),
+    # turn our "eternal" date time into symbols
+    (r'Tue Jun 18 16:13:00 2019 +0000', '🗓🕓'),
+    (r'2019-06-18 16:13', '🗓🕓'),
```

Comment:
I see two reasons why the regexp would not work:
- The plus symbol needs to be escaped (otherwise it's "space, one or more, followed by 0000")
- The time zone can not be taken for granted, the config sets git author & committer date to `2019-06-18T16:13:00`, and for me this produces run records with git log saying `Tue Jun 18 16:13:00 2019 +0200` (I'm in CEST).

I'm also adding a space for IMHO better aesthetics.


```suggestion
    (r'Tue Jun 18 16:13:00 2019 \+\d{4}', '🗓 🕓'),
    (r'2019-06-18 16:13', '🗓 🕓'),
```